environment-dev.yml

```diff
@@ -0,0 +1,58 @@
+# To set up a development environment using conda do:
```

Comment:
```suggestion
# To use:
```
is what numpy has. If you want to keep the development environment, I'd suggest changing "do" to ", use"

Reply:
I've chosen "run" :smile: